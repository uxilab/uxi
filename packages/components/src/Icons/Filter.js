import React from 'react';
import PropTypes from 'prop-types';
import SvgIcon from '../SvgIcon';
/* eslint-disable max-len */

const Filter = props => (
  <SvgIcon {...props}>
    <svg
      focusable="false"
      xmlns="http://www.w3.org/2000/svg"
      viewBox="0 0 15.25 17.5"
      width="24px"
      height="24px"
    >
      <g id="Layer_2" data-name="Layer 2">
        <g id="Layer_1-2" data-name="Layer 1">
          <path d="M15,1.09A2,2,0,0,0,13.26,0H2A2,2,0,0,0,.36,3.13L4.75,9.37v6a.74.74,0,0,0,.51.71l4,1.37a.67.67,0,0,0,.24,0,.77.77,0,0,0,.44-.14.74.74,0,0,0,.31-.61V9.42l4.59-6.26A2,2,0,0,0,15,1.09ZM6.25,14.84v-5H8.77V15.7ZM13.66,2.27,9.17,8.38H5.89L1.59,2.26a.46.46,0,0,1,0-.5A.46.46,0,0,1,2,1.5H13.26a.49.49,0,0,1,.4.77Z" />
        </g>
      </g>
    </svg>
  </SvgIcon>
);

Filter.propTypes = {
  color: PropTypes.string,
};

Filter.defaultProps = {
  color: '#6d6d71',
};

export default Filter;
