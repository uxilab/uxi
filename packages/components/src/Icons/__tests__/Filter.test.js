import React from 'react';
import Filter from '../Filter';

describe('Icon : <Filter />', () => {
  it('match snapshot', () => {
    expect(shallow(<Filter />)).toMatchSnapshot();
  });
});

