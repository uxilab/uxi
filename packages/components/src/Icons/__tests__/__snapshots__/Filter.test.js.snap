// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon : <Filter /> match snapshot 1`] = `
<SvgIcon
  color="#6d6d71"
  hoverColor="#6d6d71"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  size={24}
  style={Object {}}
  viewBox="0 0 24 24"
>
  <svg
    focusable="false"
    height="24px"
    viewBox="0 0 15.25 17.5"
    width="24px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      data-name="Layer 2"
      id="Layer_2"
    >
      <g
        data-name="Layer 1"
        id="Layer_1-2"
      >
        <path
          d="M15,1.09A2,2,0,0,0,13.26,0H2A2,2,0,0,0,.36,3.13L4.75,9.37v6a.74.74,0,0,0,.51.71l4,1.37a.67.67,0,0,0,.24,0,.77.77,0,0,0,.44-.14.74.74,0,0,0,.31-.61V9.42l4.59-6.26A2,2,0,0,0,15,1.09ZM6.25,14.84v-5H8.77V15.7ZM13.66,2.27,9.17,8.38H5.89L1.59,2.26a.46.46,0,0,1,0-.5A.46.46,0,0,1,2,1.5H13.26a.49.49,0,0,1,.4.77Z"
        />
      </g>
    </g>
  </svg>
</SvgIcon>
`;
